package.json.orig

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
Looks like this is a git comparison diff file? Could you remove it so it doesn't get committed